Ginger/GingerCoreNET/Run/RunsetActions/RunSetActionGenerateTestNGReport.cs

```diff
@@ -88,8 +89,18 @@ public override void Execute(ReportInfo RI)
         //TODO: move to Run SetAction
         private void SaveBFResults(ReportInfo RI, string folder, bool statusByGroupActivity)
         {
+            if (DynamicParameters.Count > 0)
+            {
+                ValueExpression VE = new ValueExpression(null, null);
+                int i = 0;
+                foreach (ActInputValue actInput in DynamicParameters)
+                {
```

Comment:
No need for i when using foreach.

Simply actInputValue.ValueForDriver=... 
should work